.editorconfig

```diff
@@ -89,207 +89,646 @@ dotnet_naming_rule.private_const_fields_ignore_camel_case_leading_underscore.sev
-dotnet_diagnostic.SA1109.severity = none            # Block statements should not contain embedded regions.
-dotnet_diagnostic.SA1110.severity = none            # Opening parenthesis or bracket should be on declaration line.
-dotnet_diagnostic.SA1111.severity = none            # Closing parenthesis should be on line of last parameter.
-dotnet_diagnostic.SA1112.severity = none            # Closing parenthesis should be on line of opening parenthesis.
-dotnet_diagnostic.SA1113.severity = none            # Comma should be on the same line as previous parameter.
-dotnet_diagnostic.SA1114.severity = none            # Parameter list should follow declaration.
-dotnet_diagnostic.SA1115.severity = none            # Parameter should follow comma .
-dotnet_diagnostic.SA1116.severity = none            # Split parameters should start on line after declaration.
-dotnet_diagnostic.SA1117.severity = none            # Parameters should be on same line or separate lines.
-dotnet_diagnostic.SA1118.severity = none            # The parameter spans multiple lines.
-dotnet_diagnostic.SA1119.severity = none            # Statement should not use unnecessary parenthesis.
-dotnet_diagnostic.SA1120.severity = none            # Comments should contain text.
-dotnet_diagnostic.SA1121.severity = none            # Use built-in type alias
-dotnet_diagnostic.SA1122.severity = none            # Use string.Empty for empty strings.
-dotnet_diagnostic.SA1124.severity = none            # Do not use regions.
-dotnet_diagnostic.SA1125.severity = none            # Use shorthand for nullable types.
-dotnet_diagnostic.SA1126.severity = none            # Prefix calls correctly.
-dotnet_diagnostic.SA1127.severity = none            # Generic type constraints should be on their own line.
-dotnet_diagnostic.SA1128.severity = none            # Generic type constraints should be on their own line.
-dotnet_diagnostic.SA1129.severity = none            # Do not use default value type constructor.
-dotnet_diagnostic.SA1130.severity = none            # Use lambda syntax.
-dotnet_diagnostic.SA1131.severity = none            # Use readable conditions.
-dotnet_diagnostic.SA1132.severity = none            # Each field should be declared on its own line
-dotnet_diagnostic.SA1133.severity = none            # Do not combine attributes.
-dotnet_diagnostic.SA1134.severity = none            # Attributes should not share line.
-dotnet_diagnostic.SA1135.severity = none            # Using directives should be qualified.
-dotnet_diagnostic.SA1136.severity = none            # Enum values should be on separate lines.
-dotnet_diagnostic.SA1137.severity = none            # Elements should have the same indentation.
-dotnet_diagnostic.SA1138.severity = none            # Indent elements correctly.
-dotnet_diagnostic.SA1139.severity = none            # Use literal suffix notation instead of casting.
+# AsyncFixer rules
+dotnet_diagnostic.AsyncFixer01.severity = silent # Check later as warning.
```

Comment:
Use `suggestion` instead of `silent` everywhere. That way, the violations will be visible but not disruptive.